lib/Infer/Pruning.cpp

```diff
@@ -14,7 +14,7 @@
 
 #include "souper/Infer/AbstractInterpreter.h"
 #include "souper/Infer/Pruning.h"
-
+#include "souper/Extractor/Candidates.h"
```

Comment:
what necessitated this change?

Reply:
Added a getSetSize declaration in that header.